siddpubs-journal.bib

```diff
@@ -17,6 +19,26 @@ @string{auro
 @string{ral  = "{IEEE} Robotics and Automation Letters"}
 @string{tro  = "{IEEE} Transactions on Robotics"}
 @string{thri = "{ACM} Transactions on Human-Robot Interaction"}
+@string{jmlr = "Journal of Machine Learning Research"}
+
+
+
+@article{roulet2025ilqriddp,
+  title={On Global and Local Convergence of Iterative Linear Quadratic Optimization Algorithms for Discrete Time Nonlinear Control}, 
+  author={Vincent Roulet and Srinivasa, S.S. and Maryam Fazel and Zaid Harchaoui},
```

Comment:
It looks like your full name is used in other articles: https://github.com/personalrobotics/pubs/blob/a5764d257745473e12938e4270c309934e20d93d/siddpubs-journal.bib#L53

Do you want to do that here as well for consistency? I am fine with either approach.